### ISSUE TITLE
Compact groups

### ISSUE THREAD

**Issue by QuentinRoy** [Galooshi/import-js#454]
I like the option to automatically group my imports by categories, but I am not a big fan of the whitespaces between groups. I haven't find a way to strip them and keep my imports grouped but compact. I wish we could have an option for this.

**Comment by trotzig** [comment]
I agree that the current behavior is a little inflexible. Can you explain what you want the imports to look like? We've discussed making the groups configurable in the past and I think it's mostly waiting for someone to write a PR. I haven't seen whitespace removal come up before, but I'm sure you have good reasons.

**Comment by QuentinRoy** [comment]
Simple, instead of this:

```js
import React, { Component } from 'react';
import propTypes from 'prop-types';

import { ContextSwitch, Context, contextPropType } from 'reacolo';

import ControlApp from './control/ControlApp';
import LoadingView from './LoadingView';
import XpApp from './display/XPApp';

class BaseApp extends Component {
  constructor(props) {
    super(props);
    /* stuff */
  }

  render() {
    return <div>stuff</div>;
  }
 }
```

I prefer this

```js
import React, { Component } from 'react';
import propTypes from 'prop-types';
import { ContextSwitch, Context, contextPropType } from 'reacolo';
import ControlApp from './control/ControlApp';
import LoadingView from './LoadingView';
import XpApp from './display/XPApp';

class BaseApp extends Component {
  constructor(props) {
    super(props);
    /* stuff */
  }

  render() {
    return <div>stuff</div>;
  }
 }
```

It might look less readable at with little below, but I don't find myself reading the import soon and while I like sorting the imports per categories, I don't feel it benefits much to add new lines between this categories. However I find it makes it harder to find the separation between the imports and the rest of the code (which for me is important).
Someone might also want to define what to separate or not, e.g. I could do with local import / global import, but the separation between named import and default import does not make much sense to me (in particular as they can be mixed, c.f. the first line).
My suggestion would actually to treat "grouping" and "alphabetical sort" as two different ways of sorting the imports: I find having two properties for this a bit confusing, what if I put "grouping" and "sorting" both to true ? Then deal with new line insertions as a separate option.

**Comment by trotzig** [comment]
Thanks for providing that context. I agree that it's a little odd to have both those configuration options. I think the reason is that they were added one by one gradually, and we never took a holistic look at things. 

Providing a configuration option to disable the whitespace shouldn't be too hard I think. If that's something you'd find useful I'll be happy to review a PR.
